handlers/api.go

```diff
@@ -1507,7 +1522,7 @@ func ApiValidatorDailyStats(w http.ResponseWriter, r *http.Request) {
 			sendErrorResponse(w, r.URL.String(), "invalid start_day parameter")
 			return
 		}
-		if start < endDay {
+		if start > endDay {
```

Comment:
This was a bugfix unrelated to the issue that prevented you from entering any "start" value at all.